docs/advanced/releases.md

```diff
@@ -76,23 +106,29 @@ Release Candidates will be tested in a production server (usually meta-decidim)
+        1. Remove all files except `.git` from clone A.
+        1. Copy all files except `.git` from clone B.
+        1. Paste all files except `.git` to clone A.
+        1. Commit the differences to be merged to `master`: `git add -A && git commit -m "Merge release/x.y-stable"`
     1. `git push origin release/x.y.z`
     1. Create the PR. The base for this PR should be `master`, but GitHub may crash if there are a lot of changes. As a workaround create the branch against `develop` and, when created, change the base to `master`.
     1. Still don't merge it.
 1. Before merging the PR to upgrade `master`, check that the stable branch for the previous version exists. For instance, if we are going to release v0.22.0, there should be a `release/0.21-stable` branch in the repository. If such branch does not exists, it has to be created now, before merging the new release. So, if this is the release of v0.22.0, branch off `release/0.21-stable` from `master`. These stable branches will be able to receive bugfixes, backports and will be the origin of patch releases for older releases.
 1. Merge (after proper peer review) the PR to `master` and remove `release/x.y.z` branch.
-1. Run `git checkout master && bin/rake release_all`, this will create all the tags, push the commits and tags and release the gems to RubyGems.
+1. Run `git checkout master && git pull && bin/rake release_all`, this will create all the tags, push the commits and tags and release the gems to RubyGems.
 1. Once all the gems are published you should create a new release at this repository, just go to the [releases page](https://github.com/decidim/decidim/releases) and create a new one.
-1. Create the stable branch for the current version. From `master`: `git checkout -b release/x.y-stable && git push origin release/x.y-stable`.
+1. Create the stable branch for the current version if it does not exist. From `master`: `git checkout -b release/x.y-stable && git push origin release/x.y-stable`.
 1. Update Decidim's Docker repository as explained in the Docker images section below.
 1. Update Crowdin synchronization configuration with Github:
     1. Add the new `release/x.y-stable` branch.
     1. Remove from Crowdin branches that are not officially supported anymore. That way they don't synchronize with Github.
-1. Update the `CHANGELOG.MD` in `release/x.y-stable`. At the top you should have an `Unreleased` header with the `Added`, `Changed`, `Fixed` and `Removed` empty sections. After that, the header with the current version. Add the `Unreleased` section or create the new current version section.
+1. Update the `CHANGELOG.MD` in `release/x.y-stable`. At the top you should have an `Unreleased` header with the `Added`, `Changed`, `Fixed` and `Removedgit` empty sections. After that, the header with the current version. Add the `Unreleased` section or create the new current version section.
```

Comment:
`Removedgit` seems like the typical error where you have too many windows opened. 

I catch these kinds of mistakes by using `git add -p`

Reply:
Fixed, thanks!